elasticdl/python/master/servicer.py

```diff
@@ -83,3 +100,22 @@ def report_version(self, request, _):
                 master_locking=False, model_version=request.model_version
             )
         return empty_pb2.Empty()
+
+    def get_average_task_complete_time(self):
+        if len(self._task_complete_times) < 20:
+            return {
+                elasticdl_pb2.TRAINING: 36000,
+                elasticdl_pb2.EVALUATION: 36000,
+            }
+        else:
+            return {
+                elasticdl_pb2.TRAINING: statistics.mean(
+                    self._task_complete_times[elasticdl_pb2.TRAINING]
+                ),
+                elasticdl_pb2.EVALUATION: statistics.mean(
+                    self._task_complete_times[elasticdl_pb2.EVALUATION]
+                ),
+            }
+
+    def get_worker_liveness_time(self, worker_id):
```

Comment:
`get_worker_liveness_time` may be called before `self._worker_liveness_time[worker_id]` is set.
Init `self._worker_liveness_time[worker_id]` in `get_task` if not already initialized.

Reply:
Also, we may need to update `self._worker_liveness_time` in `get_task`, as the worker may receive `WAIT` task and doing nothing for a while, then get an evaluation task. This task may timeout as `self._worker_liveness_time` is not updated for a long time.

Reply:
Done. Update the liveness time inside get_task